src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
```suggestion
  hide() {
    this.hidden = !this.hidden;
  }
```
Зараз цей метод завжди буде встановлювти hidden в true, а за умовами задачі, цей метод повинен "перемикати" значення на протилежне
>Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores.